.github/workflows/CI.yaml

```diff
@@ -283,3 +283,125 @@ jobs:
+
+      - name: "Install: Node.js"
+        uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+
+      - name: "Install: Node.js packages"
+        run: ./scripts/gear.sh init js
+
+      - name: "Install: Build deps"
+        run: |
+          sudo apt update
+          sudo apt install -y git clang curl libssl-dev llvm libudev-dev cmake
+
+      - name: "Install: grcov"
+        run: |
+          curl https://api.github.com/repos/mozilla/grcov/releases/latest | 
+          grep -wo "https.*x86_64.*linux-gnu.*tar\.bz2" | 
+          xargs curl -L | 
+          tar xj
+          
+      - name: "Install: rust-covfix"
+        run: |
+          curl -L https://github.com/Kogia-sima/rust-covfix/releases/download/v0.2.2/rust-covfix-linux-x86_64.tar.xz | tar Jxf -
+          mv rust-covfix-linux-x86_64/rust-covfix ./
+
+      - name: "Cache: Unpack"
+        if: github.event.pull_request.merged == false
+        continue-on-error: true
+        run: |
+          cp /root/cache/gear-test* /tmp/
+          tar -xf /tmp/gear-test_cache.tar -C /
+          tar -xf /tmp/gear-test_cargo_registry.tar -C /
+          tar -xf /tmp/gear-test_target.tar
+
+      - name: "Build: Split examples by .opt and .meta"
+        run: ./scripts/gear.sh build wasm-proc
+
+      - name: "Build: Examples (WASM)"
+        run: ./scripts/gear.sh build examples --locked
+
+      - name: "Build: Split examples by .opt and .meta"
+        run: ./scripts/gear.sh build examples-proc
+        
+      - name: "Test: Specified packages"
+        run: >-
+          cargo nextest run 
+          -p gear-core 
+          -p gear-core-processor 
+          -p gear-backend-sandbox 
+          -p gear-backend-wasmtime 
+          -p gear-runtime 
+          -p gear-runtime-interface 
+          -p gear-lazy-pages 
+          -p pallet-gear-debug 
+          -p pallet-gas 
+          -p pallet-gear-program 
+          -p pallet-usage 
+          --no-fail-fast --tests
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
+        
+      - name: "Test: Process node runtime testsuite"
+        run: ./scripts/gear.sh coverage rtest
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
+          
+      - name: "Test: Process gear-test"
+        run: ./scripts/gear.sh coverage gtest
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
+            
+      - name: "Coverage: Aggregate"
+        run: >-
+          ./grcov . --binary-path ./target/debug/ -s . -t lcov --branch --ignore-not-existing
+          --ignore "/*"
+          --ignore "examples/*"
+          --ignore "utils/*"
+          --ignore "target/*"
+          --ignore "gear-test/*"
+          -o ./lcov.info
+        
+      - name: "Coverage: Fix report"
+        run: ./rust-covfix -o lcov.info lcov.info
+        
+      - name: "Coverage: Publish"
+        uses: codecov/codecov-action@v1
+        with:
+          file: ./lcov.info
+
+      - name: "Cache: Pack"
```

Comment:
I think packing cache from code coverage is not needed

Reply:
Do we need unpacking then?

Reply:
yes, as it will speed up the build time